CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ file(GLOB SOURCE_FILES
         src/univalue/include/*.h
         src/univalue/lib/*.cpp
         src/univalue/lib/*.h
+        src/util/*.h
+        src/util/*.cpp
         src/wallet/*.cpp
         src/wallet/*.h
         src/wallet/test/*.cpp
```

Comment:
`src/wallet/test/*.h` due to `./wallet/test/wallet_test_fixture.h`

Also missing:
```
src/bench/data/*.h                                                              
src/config/*.h
src/logging/*.h
src/node/*.cpp
src/node/*.h
src/obj/*.h
src/support/allocators/*.h
src/support/*.cpp
src/support/*.h
src/test/fuzz/*.cpp
src/test/fuzz/*.h
src/test/util/*.cpp
src/test/util/*.h
src/univalue/gen/*.cpp
src/univalue/test/*.cpp
src/util/*.cpp
src/util/*.h
```

Should we include 3rd party libraries?
If so, missing also:
```
src/crc32c/include/crc32c/*.h
src/crc32c/src/*.h
src/crypto/ctaes/*.h
src/dashbls/src/*.cpp                                                           
```

I used this script to gather filelist:
```
find src/ -name '*.h' -or -name '*.cpp'    | sed -r 's/[^/]*.(h|cpp)$/*.\1/'  | sort | uniq > list.txt
```

Reply:
Ooops, I am little late.
Updated comment below.

Btw, there's still missing files:
```
src/univalue/test/*.cpp                                                         
src/wallet/test/*.h
src/univalue/gen/*.cpp
```

(same script)

should we include 3rd party libraries also?
```
src/crc32c/include/crc32c/*.h
src/crc32c/src/*.h
src/crypto/ctaes/*.h
src/dashbls/src/*.cpp  
src/secp256k1/include/*.h

Reply:
I think it's good to add 3rd party libs as well